.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
This looks like specific editor settings? Those should not be included.

Reply:
It's not: https://editorconfig.org/

I'd think it would help with consistency of code no matter what editor or IDE is used.